geopy/geocoders/maptiler.py

```diff
@@ -0,0 +1,189 @@
+from geopy.compat import quote, urlencode
+from geopy.geocoders.base import DEFAULT_SENTINEL, Geocoder
+from geopy.location import Location
+from geopy.point import Point
+from geopy.util import logger
+
+__all__ = ("MapTiler", )
+
+
+class MapTiler(Geocoder):
+    """Geocoder using the MapTiler API.
+
+    Documentation at:
+        https://cloud.maptiler.com/geocoding/?_ga=2.134043661.283487034.1581321159-amp-OxJ6mv8GSFOKBYaTzxmOkQ
```

Comment:
Could you please remove the `_ga` cookie?

Also, the link doesn't seem to be working without registration/authentication. Is there a publicly available doc which doesn't require authentication?

Reply:
sadly I could not find any publicly available docs